### DISCUSSION TITLE
Image component wrapper styling

### DISCUSSION THREAD

**Discussion by MathiasKandelborg** [vercel/next.js#18457] [Ideas]
I cannot close this. Problem solved with the `layout` prop introduced in 10.0.1-canary.1.

It's live with v10.0.1

**Comment by M1ck0** [comment]
You can always use `unsize` prop that won't change size and width and will server image as-is. 

Or you can play with CSS selectors and set styling to image. 

I didn't have these problems so if you could provide code for `<Image/>` component I could help you.

**Reply by MathiasKandelborg** [reply]
I can not use the `unsized` prop. That would decrease the lighthouse score because the image does not have a set width.

The `unsized` prop is not the solution in this case. It's the additional wrappers created around the `img` element that has hard-coded styling. If there were no added styling, there wouldn't be a problem, I've tried removing the width wrapper manually, and it works like expected/before.

Here's the code: https://github.com/MathiasKandelborg/TRUE-framework/blob/alpha/components/UI/Pages/Home/HomePage.tsx#L28-L36

I was using srcset before; you can look at the accompanying file for styling. It worked with '100%' width on the native img component, changing the styling on the image makes no difference now (because of the wrappers).

**Reply by MathiasKandelborg** [reply]
This is supposed to be a priority image, but I haven't gotten the component working properly, so I've not added it yet.

This is where the hardcoded styling happens: https://github.com/vercel/next.js/blob/canary/packages/next/client/image.tsx#L280-L295

**Comment by agcty** [comment]
There's a similar discussion here: https://github.com/vercel/next.js/discussions/18739, basically it's impossible to do what you're trying to do at least from what I've encountered

**Reply by MathiasKandelborg** [reply]
Oh, I forgot to close this. The new layout prop solves the issue. It is entirely possible, and you can now choose other options besides the default styling.

It's still in canary, though.

**Reply by timneutkens** [reply]
It's on 10.0.1 👍

**Reply by MathiasKandelborg** [reply]
Oh, one more time :sweat_smile: Didn't see 10.0.1 was released! :rocket:

**Comment by MathiasKandelborg** [comment]
New layout prop fixes this.

**Comment by agcty** [comment]
This one seems to be correlated: https://github.com/vercel/next.js/discussions/18739
